elastic/kibana
# [Infra UI] Waffle Map

This PR was closed in favor of elastic/kibana#21871 due to the VS Code/Git/Merge snafu

---

This PR adds the basic functionality for the Waffle Map to the Infra UI. Currently there is nothing configurable via the UI. The options are hard coded in the `withMap` container query, this will change as the UI features are developed to support it. This PR is based on elastic/kibana#21504 and includes changes in the diff.

![image](https://user-images.githubusercontent.com/41702/43792307-1949e7ae-9a2d-11e8-9e61-748a53b578e4.png)

How to test in browser:

1. Point Kibana at the `demo-stack` Kibana instance via `config/kibana.dev.yml` (reach out to @simianhacker for actual configuration) 
2. Click on infra icon. You should see something very close to above.
